apps/src/code-studio/components/aiTutor/aiTutorChatMessagesTable.tsx

```diff
@@ -1,10 +1,20 @@
+import {getSectionAITutorInteractions} from '@cdo/apps/aiTutor/interactionsApi';
 import React from 'react';
 
 /**
  * Renders a table showing the section's students' chat messages with AI Tutor.
  */
+interface AITutorChatMessagesTableProps {
+  sectionId: number;
+}
 
-const AITutorChatMessagesTable: React.FunctionComponent = () => {
+const AITutorChatMessagesTable: React.FunctionComponent<
+  AITutorChatMessagesTableProps
+> = async ({sectionId}) => {
+  const aiTutorInteractions = await getSectionAITutorInteractions(sectionId);
+
+  console.log('sectionId: ', sectionId);
+  console.log('aiTutorInteractions ', aiTutorInteractions);
```

Comment:
The data that we're getting back from the api looks like this: 
<img width="1097" alt="Screenshot 2024-02-20 at 8 37 53 PM" src="https://github.com/code-dot-org/code-dot-org/assets/12300669/1282276c-0538-425c-a9b6-c6be03c1c834">
<img width="555" alt="Screenshot 2024-02-20 at 8 38 16 PM" src="https://github.com/code-dot-org/code-dot-org/assets/12300669/43f1afc2-5426-4e59-b0b7-e6d27d28fb45">

We're going to need a way to:
-  get the user's name from their id, 
- format the date and time correctly 
- and strip the system_prompt from the prompt field so we're left with just the student's question.  (Do we internally need to know what the system prompt was at the time the question was asked?)

Reply:
Similarly, confirm with Product:
- what we want to show for validation and compilation scenarios
- how we want to indicate inappropriate conversations